src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,96 @@
+package parser;
+
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.ListCommand;
+import command.MarkCommand;
+import command.SearchCommand;
+import command.UnmarkCommand;
+import fridayException.FridayException;
+import fridayException.InvalidDeadlineArgument;
+import fridayException.InvalidDeleteArgument;
+import fridayException.InvalidEventArgument;
+import fridayException.InvalidFindArgument;
+import fridayException.InvalidFridayCommand;
+import fridayException.InvalidMarkArgument;
+import fridayException.InvalidSearchArgument;
+import fridayException.InvalidTodoArgument;
+import fridayException.InvalidUnmarkArgument;
+import task.Deadline;
+import task.Event;
+import task.ToDo;
```

Comment:
Well done with importing the specific classes you needed, instead of just importing * from these packages! 👍